app/controllers/stats_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Controller for the stats endpoint
+class StatsController < ApiController
+  skip_authorization_check only: [:index]
+
+  # GET /stats
+  # only returns json
+  def index
+    result = StatsController.fetch_stats(current_user)
+
+    # Simplify the response, by mutation. Show only ids from active record models.
+    result[:recent].transform_values! do |query|
+      query.pluck(:id)
+    end
+
+    built_response = Settings.api_response.build(:ok, result)
+    render json: built_response, status: :ok, layout: false
+  end
+
+  def self.fetch_stats(current_user)
+    online_window = 2.hours.ago
+    {
+      summary: {
+        users_online: User.recently_seen(online_window).count,
+        users_total: User.count,
+        online_window_start: online_window,
+        annotations_total: AudioEvent.count,
+        annotations_total_duration: AudioEvent.total_duration_seconds,
+        annotations_recent: AudioEvent.recent_within(1.month.ago).count,
+        audio_recording_total: AudioRecording.count,
+        audio_recording_recent: AudioRecording.created_within(1.month.ago).count,
+        audio_recording_total_duration: AudioRecording.total_duration_seconds,
+        audio_recording_total_size: AudioRecording.total_data_bytes.to_i,
+        tags_total: Tag.count,
+        tags_applied_total: Tagging.count
+      },
+      recent: {
+        audio_recordings: Access::ByPermission.audio_recordings(current_user).most_recent(10),
+        audio_events: Access::ByPermission.audio_events(current_user).most_recent(10)
```

Comment:
When the `user` is `nil` the query will still happen. Then the `ByPermission` module will only show public recordings.

This is tested in specs with the `create_anon_heierarchy` factory helper.

Why 10? Because that is what it was, and we're retaining backward compat with the view. Also, this is not a standard CRUD API endpoint. It returns a freeform structure for a virtual model for which there is only one item.